### DISCUSSION TITLE
How tu run git commands?

### DISCUSSION THREAD

**Discussion by manzomium** [adieyal/sd-dynamic-prompts#158] [Q&A] [answered]
Hello

I'm posting this here, hoping to get some help. I'm new to github, notebooks, etc.
I read in the installation instructions this:
You can also install it manually by **running the following command from within the webui directory**:

git clone https://github.com/adieyal/sd-dynamic-prompting/ extensions/dynamic-prompts

how do you do that? I tried with the Windows cmd but I wasn't successful. Thank you for pointing me to any tutorials! I tried to search on google but got lost soon...

**Comment by adieyal** [comment]
the easiest way to install extensions for auto1111 is within the ui itself in the extensions tab - before trying git, you should try that first.

**Reply by manzomium** [reply]
thanks! this is what I'll be doing. However, I'd like to be able to do it also the other way, as it seems it's useful knowledge. Any tips? Thanks a lot!

**Comment by manzomium** [comment, marked as answer]
UPDATE: I run into an error and cannot load the extension;

Error completing request
Arguments: ('', 'https://github.com/adieyal/sd-dynamic-prompting/extensions/dynamic-prompts') {}
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/call_queue.py", line 45, in f
    res = list(func(*args, **kwargs))
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui_extensions.py", line 140, in install_extension_from_url
    repo = git.Repo.clone_from(url, tmpdir)
  File "/usr/local/lib/python3.8/dist-packages/git/repo/base.py", line 1148, in clone_from
    return cls._clone(git, url, to_path, GitCmdObjectDB, progress, multi_options, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/git/repo/base.py", line 1086, in _clone
    finalize_process(proc, stderr=stderr)
  File "/usr/local/lib/python3.8/dist-packages/git/util.py", line 386, in finalize_process
    proc.wait(**kwargs)
  File "/usr/local/lib/python3.8/dist-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git clone -v https://github.com/adieyal/sd-dynamic-prompting/extensions/dynamic-prompts /content/gdrive/MyDrive/sd/stable-diffusion-webui/tmp/dynamic-prompts
  stderr: 'Cloning into '/content/gdrive/MyDrive/sd/stable-diffusion-webui/tmp/dynamic-prompts'...
remote: Not Found
fatal: repository 'https://github.com/adieyal/sd-dynamic-prompting/extensions/dynamic-prompts/' not found
'

**Reply by adieyal** [reply]
I'll take a look in a few hours

**Reply by manzomium** [reply]
Thanks a lot! I've posted a new issue [here](https://github.com/adieyal/sd-dynamic-prompts/issues/159#issuecomment-1374551873) so maybe you can cose this one :-)
